mk/submodule.mk

```diff
@@ -8,16 +8,21 @@ ifeq ($(UNAME_S),Darwin)
     # macOS
     UNAME_S := darwin
 endif
+# For de10-nano and arrow sockit board
```

Comment:
Append `Raspberry Pi 2/3`

Reply:
The machine hardware name of Raspberry Pi 2/3 are `armv7l`, too?

If so, on `arm`-`Linux`, the `libtuv` supports RPi2 but not RPi3.
We might include RPi2 first and modified the `docs/build-n-test.md` as well?

Reply:
RPi 2 is based on Arm Cortex-A7. RPi 3 is ARMv8-A architecture with 32-bit Linux (so does `armv7l`).

Reply:
Maybe @wusyong can verify RPi2 as well.

Reply:
I have three board but none of them are RPi2. They are 1B, 1B+, and Pi zero respectively. The result of `uname -m` is `armv6l`.

Reply:
No problem at all. @ajblane has RPi 3 board.

Reply:
Currently, OPTEE and Linux run on RPi 3 board and there is no gcc compiler in Linux. I need to build cross compiling for dcurl with libtuv.

Reply:
`Raspberry Pi 2/3` has been added in the Makefile and the document.
The `libtuv` is also modified for supporting `Raspberry Pi 3` on `arm`-`Linux`.